src/core/seed.js

```diff
@@ -62,99 +82,19 @@ Seed.prototype.to_json = function() {
     return NaN;
   }
 
-  const output = Base.encode_check(Base.VER_FAMILY_SEED, this.to_bytes());
+  const output = Base.encode_check(this._version, this.to_bytes());
 
   return output;
 };
 
-function append_int(a, i) {
-  return [].concat(a, i >> 24, (i >> 16) & 0xff, (i >> 8) & 0xff, i & 0xff);
-}
-
-function firstHalfOfSHA512(bytes) {
-  return sjcl.bitArray.bitSlice(
-    sjcl.hash.sha512.hash(sjcl.codec.bytes.toBits(bytes)),
-    0, 256
-  );
-}
-
-// Removed a `*` so this JSDoc-ish syntax is ignored.
-// This will soon all change anyway.
-/*
-* @param account
-*        {undefined}                 take first, default, KeyPair
-*
-*        {Number}                    specifies the account number of the KeyPair
-*                                    desired.
-*
-*        {Uint160} (from_json able), specifies the address matching the KeyPair
-*                                    that is desired.
-*
-* @param maxLoops (optional)
-*        {Number}                    specifies the amount of attempts taken
-*                                    to generate a matching KeyPair
-*
-*/
-Seed.prototype.get_key = function(account, maxLoops) {
-  let account_number = 0, address;
-  let max_loops = maxLoops || 1;
-
+Seed.prototype.get_key = function() {
   if (!this.is_valid()) {
```

Comment:
Arguably should throw an error here if there's any arguments, to let people know things have changed.